framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java

```diff
@@ -169,6 +170,22 @@ public boolean valid() {
       return false;
     }
 
+    int maxByteSize = 200;
```

Comment:
Is a maximum byte size of 200 too large? Can it be set smaller?

Reply:
The target of method `valid` is to make sure that log's content is not too large, so the value of  maxByteSize  is not very important. The length of address may be 42 bytes or more (not certain), sig is 65, codeVersion may not stable(20 bytes?). Can you give a suggestion?

Reply:
Just take the maximum of the three values, for example 65.

Reply:
In fact, I don't test the true length of address ang sig, because i have no fastforword node to use. Only give a theoretical value. Give an upper value can be ok, but give an exact value is not necessary, because method checkHelloMessage will verify the address and sig again. The main purpose is to not record too many message in log file.